test/core/PriceManager.js

```diff
@@ -74,48 +64,27 @@ describe("PriceManager", function () {
         feeRewardBasisPoints = 70000 // FeeRewardBasisPoints 70%
         depositFee = 3000
         stakingFee = 3000
-        vaultPriceFeed = await deployContract("VaultPriceFeed", [])
+
         priceManager = await deployContract("PriceManager", [
-          vaultPriceFeed.address
+          pyth
```

Comment:
which value is this pyth?

Reply:
for testing, we cant pass pyth. pyth provide only for mainnet and test networks, not for hardhat test.
so I left it as empty for now.